getting_started/scripting/gdscript/gdscript_documentation_comments.rst

```diff
@@ -0,0 +1,217 @@
+The tag must be at the beginning of a line (ignoring preceding white space) and has
+the format ``@``, followed by the keyword and finishing with a colon.
+
+Tags
+~~~~
+
++-------------------+--------------------------------------------------------+
+| Brief description | No tag and lives at the very beginning of  |
+|                   | the documentation section.                             |
++-------------------+--------------------------------------------------------+
+| Description       | ``@desc:``                                             |
+|                   |                                                        |
++-------------------+--------------------------------------------------------+
+| Tutorial          | ``@tutorial[( The Title Here )]:``                     |
+|                   |                                                        |
++-------------------+--------------------------------------------------------+
+
+**Example:**
+
+::
+
+    extends Node2D
+	
+    ## 
+    ## A brief description of your script.
+    ##
+    ## @desc:
+    ##     A more detailed description of the script.
+    ## 
+    ## @tutorial:            http://the/tutorial1/url.com
+    ## @tutorial(Tutorial2): http://the/tutorial2/url.com
+    ##
+
+.. warning:: If there is any space in between the tag name and colon, for example 
+             ``@desc  :``, it won't treated as a valid tag and will be ignored.
+
+
+.. note:: When the description is multi line, the preceding and trailing white
+          spaces will be striped and joined with a single space. To preserve the line 
+          break (or any other alignment) use
+          :ref:`bbcode <doc_bbcode_in_richtextlabel>`.
+
+Documenting script members
+--------------------------
+
+Documentation of a script member must immediately precede the member or its
+annotations if it has any. The exception to this is enum values whose description should
+be on the same line as the enum for readability. 
+The description can have more than one line but every line must start
+with the double hash symbol ``##`` to be considered as part of the documentation.
+The script documentation will update in the editor help window every time the
+script is updated. If any member variable or function name starts with an
+underscore it will be treated as private. It will not appear in the documentation and
+will be ignored in the help window.
+
+Members that are applicable for the documentation
+
+- Inner class
+- Constant
+- Function
+- Signal
+- Variable
+- Enum
+- Enum value
+
+Examples
+--------
+
+::
+
+    extends Node2D
+	
+    ## 
+    ## A brief description of your script.
+    ##
+    ## @desc:
+    ##     The description of the script, what it
+    ##     can do, and any further detail.
+    ## 
+    ## @tutorial:            http://the/tutorial1/url.com
+    ## @tutorial(Tutorial2): http://the/tutorial2/url.com
+    ##
+    
+    ## The description of the variable v1
```

Comment:
Always end descriptions with a period for consistency with the official class reference:

```suggestion
    ## The description of the variable v1.
```